server/container_stop.go

```diff
@@ -25,6 +25,9 @@ func (s *Server) StopContainer(ctx context.Context, req *pb.StopContainerRequest
 		if err := s.runtime.StopContainer(c, req.Timeout); err != nil {
 			return nil, fmt.Errorf("failed to stop container %s: %v", c.ID(), err)
 		}
+		if err := s.storageRuntimeServer.StopContainer(c.ID()); err != nil {
```

Comment:
we still need to keep that in both places because remove pod sandbox still needs to stop containers on demand https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/cri/v1alpha1/runtime/api.proto#L34-L36

Btw, I think sameo was referring to the fact that in container_remove we were still calling StopContainer regardless of status, now in container_remove we call stop_container only if status tells us to do so see here https://github.com/kubernetes-incubator/cri-o/pull/530/files#diff-a7484af4a1e8a5664e3a8829ad1872fcR30